ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -23,7 +23,8 @@ static_library('nnstreamer_decoder_direct_video',
 
 # image labeling
 decoder_sub_image_labeling_sources = [
-  'tensordec-imagelabel.c'
+  'tensordec-imagelabel.c',
+  'tensordecutil.c'
```

Comment:
Q: It's minor question but I just wonder what the file naming policy is here. `tensordecutil.c` or `tensordec-util.c` ?

Reply:
If we name it tensordec-util.c, it would look like a "util" subplugin for tensordec.

Reply:
Right. Got it.